Cargo.toml

```diff
@@ -14,118 +14,120 @@ panic = "unwind"
-sc-executor = { version = "0.32.0", default-features = false }
-sc-keystore = { version = "25.0.0", default-features = false }
-sc-network = { version = "0.34.0", default-features = false }
-sc-network-sync = { version = "0.33.0", default-features = false }
-sc-rpc-api = { version = "0.33.0", default-features = false }
-sc-rpc = { version = "29.0.0", default-features = false }
-sc-service = { version = "0.35.0", default-features = false }
-sc-sysinfo = { version = "27.0.0", default-features = false }
-sc-telemetry = { version = "15.0.0", default-features = false }
-sc-tracing = { version = "28.0.0", default-features = false }
-sc-transaction-pool-api = { version = "28.0.0", default-features = false }
-sc-transaction-pool = { version = "28.0.0", default-features = false }
-sp-api = { version = "26.0.0", default-features = false }
-sp-application-crypto = { version = "30.0.0", default-features = false }
-sp-arithmetic = { version = "23.0.0", default-features = false }
-sp-block-builder = { version = "26.0.0", default-features = false }
-sp-blockchain = { version = "28.0.0", default-features = false }
-sp-consensus-aura = { version = "0.32.0", default-features = false }
-sp-consensus-babe = { version = "0.32.0", default-features = false }
-sp-consensus = { version = "0.32.0", default-features = false }
-sp-core = { version = "28.0.0", default-features = false }
-sp-inherents = { version = "26.0.0", default-features = false }
-sp-io = { version = "30.0.0", default-features = false }
-sp-keystore = { version = "0.34.0", default-features = false }
-sp-genesis-builder = { version = "0.7.0", default-features = false }
-sp-npos-elections = { version = "26.0.0", default-features = false }
-sp-offchain = { version = "26.0.0", default-features = false }
-sp-runtime = { version = "31.0.0", default-features = false }
-sp-session = { version = "27.0.0", default-features = false }
-sp-staking = { version = "26.0.0", default-features = false }
-sp-state-machine = { version = "0.35.0", default-features = false }
-sp-std = { version = "14.0.0", default-features = false }
-sp-storage = { version = "19.0.0", default-features = false }
-sp-timestamp = { version = "26.0.0", default-features = false }
-sp-tracing = { version = "16.0.0", default-features = false }
-sp-transaction-pool = { version = "26.0.0", default-features = false }
-sp-version = { version = "29.0.0", default-features = false }
-substrate-build-script-utils = { version = "11.0.0", default-features = false }
-substrate-prometheus-endpoint = { version = "0.17.0", default-features = false }
-substrate-wasm-builder = { version = "17.0.0", default-features = false }
-xcm-builder = { package = "staging-xcm-builder", version = "7.0.0", default-features = false }
-xcm-executor = { package = "staging-xcm-executor", version = "7.0.0", default-features = false }
-xcm = { package = "staging-xcm", version = "7.0.0", default-features = false }
+cumulus-client-cli = { version = "0.14.0", default-features = false }
```

Comment:
The versions I find inside polkadot-v1.13.0 are "0.7.0"! How have you extracted these versions?